### PR TITLE
ansible-doc: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/ansible-doc.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+> Muestra una concisa lista de complementos y sus breves descripciones.
+> Más información: <https://docs.ansible.com/ansible/latest/cli/ansible-doc.html>.
+
+- Lista de complementos disponibles acorde a su acción (módulos):
+
+`ansible-doc --list`
+
+- Lista de complementos disponibles dado un tipo específico:
+
+`ansible-doc --type {{tipo_complemento}} --list`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ansible-doc.md:13]:
```suggestion
- Lista de complementos disponibles dado un tipo específico:

`ansible-doc --type {{become|cache|callback|cliconf|connection|...}} --list`
```

Mandatory fix (error source: original page): [broken placeholder content](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): expected 5 listed alternatives with ellipsis.

File: pages.es/common/ansible-doc.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Muestra información sobre un complemento acorde su acción específica (módulo):
+
+`ansible-doc {{nombre_complemento}}`
+
+- Muestra información acerca de un complemento dado un tipo específico:
+
+`ansible-doc --type {{tipo_complemento}} {{nombre_complemento}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/ansible-doc.md:21]:
```suggestion
- Muestra información acerca de un complemento dado un tipo específico:

`ansible-doc --type {{become|cache|callback|cliconf|connection|...}} {{nombre_complemento}}`
```

The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

File: pages.es/common/ansible-doc.md (hunk 1 above)
Comment (kbdharun) [reply pages.es/common/ansible-doc.md:13]: So, we should update the original file too?

Comment (EmilySeville7cfg) [reply pages.es/common/ansible-doc.md:13]: Already done in #9739. :)

**Review by EmilySeville7cfg — APPROVED** [review]

**Pull request merged**